src/cgame/cg_main.cpp

```diff
@@ -1253,9 +1334,15 @@ void CG_Init( int serverMessageNum, int clientNum, const glconfig_t& gl, const G
 
 	CG_SetMapNameFromServerinfo();
 
+	/* The cgs.mapname variable must already be set at this point. */
+	SetMapInfoFromArena( cgs.mapname );
+
 	// load the new map
 	trap_CM_LoadMap(cgs.mapname);
 
+	/* The map must already be loaded at this point. */
```

Comment:
This comment still needs to be fixed

Reply:
This was rewritten and this comment is gone.